src/sentry/eventstream/kafka/backend.py

```diff
@@ -244,7 +244,7 @@ def _build_streaming_consumer(
         initial_offset_reset: Union[Literal["latest"], Literal["earliest"]],
         strict_offset_reset: Optional[bool],
     ) -> StreamProcessor[KafkaPayload]:
-        configure_metrics(MetricsWrapper(metrics.backend))
+        configure_metrics(MetricsWrapper(metrics.backend), name="eventstream")
```

Comment:
The test failure seems relevant. I don't see an option to pass `name` to configure_metrics call

Reply:
Oops, fixed. Paren in the wrong place, the name was supposed to be passed to the MetricsWrapper.